data_validation/find_tables.py

```diff
@@ -132,8 +132,9 @@ def expand_tables_of_asterisk(
             mapping
             and mapping[consts.CONFIG_SCHEMA_NAME]
             and mapping[consts.CONFIG_TABLE_NAME] == "*"
-            and consts.CONFIG_TARGET_SCHEMA_NAME not in mapping
-            and consts.CONFIG_TARGET_TABLE_NAME not in mapping
+            # Looking for schema.* without a target side qualifier.
+            and not mapping.get(consts.CONFIG_TARGET_SCHEMA_NAME, None)
+            and not mapping.get(consts.CONFIG_TARGET_TABLE_NAME, None)
```

Comment:
Ahhh I get it now, makes sense, thanks! Only the comment already clarifies a lot, we can go back to the initial approach since it's simpler to read and understand, what do you think?

Reply:
I think we leave the new code because it is more robust.

Reply:
That's right, no prob. Maybe it's the case to add a test to guarantee we skip this scenario? Something similar to the new one `test_column_multi_table_all_config_managers`

Reply:
I've added an extra unit test. Good suggestion, thanks.